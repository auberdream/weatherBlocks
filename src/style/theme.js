export default {
  palette: {
    weather: {
      sunny: '#f72585',
      cloudy: '#b5179e',
      mist: '#7209b7',
      rain: '#560bad',
      thunder: '#3a0ca3',
      snow: '#3f37c9',
      sleet: '#4361ee',
      fog: '#4cc9f0',
    },
  },
  select: {
    background: '#FFF',
  },
};
