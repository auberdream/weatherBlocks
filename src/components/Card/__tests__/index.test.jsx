import React from 'react';
import renderer from 'react-test-renderer';
import { Grommet } from 'grommet';

import theme from '../../../style/theme';
import Card from '..';

it('Renders correctly', () => {
  const tree = renderer.create(
    <Grommet theme={theme}>
      <Card
        date="01/01/1970 12:00"
        condition="Cloudy"
        temperature="12Â°"
        rainChance="25"
        snowChance="0"
        position={{ x: 150, y: 300 }}
      />
    </Grommet>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
