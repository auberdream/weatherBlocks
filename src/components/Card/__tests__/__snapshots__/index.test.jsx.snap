// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="StyledGrommet-sc-19lkkz7-0 djOnwZ"
>
  <div
    className="StyledBox-sc-13pk1d4-0 exUXVR"
    style={
      Object {
        "left": 160,
        "position": "absolute",
        "top": 320,
      }
    }
  >
    <div
      className="StyledBox-sc-13pk1d4-0 jmuHez"
    >
      <span
        className="StyledText-sc-1sadyjn-0 bSYtvL"
      >
        01/01/1970 12:00
      </span>
      <span
        className="StyledText-sc-1sadyjn-0 cDuYNf"
      >
        12Â°
      </span>
    </div>
    <div
      className="StyledBox-sc-13pk1d4-0 iBWTSp"
    >
      <span
        className="StyledText-sc-1sadyjn-0 gdmUOw"
      >
        Cloudy
      </span>
    </div>
    <div
      className="StyledBox-sc-13pk1d4-0 dYebPD"
    >
      <span
        className="StyledText-sc-1sadyjn-0 gOVGlS"
      >
        Rain - 25%
      </span>
    </div>
    <div
      className="StyledBox-sc-13pk1d4-0 dYebPD"
    >
      <span
        className="StyledText-sc-1sadyjn-0 gOVGlS"
      >
        Snow - 0%
      </span>
    </div>
  </div>
</div>
`;
